install.sh

```diff
@@ -33,6 +36,29 @@ MAGENTA='\e[1;35m';
 RESET='\e[0m'
 
 echo -e "${GREEN}Welcome to Chromebrew!${RESET}\n"
+
+# disallow non Chrome OS devices
+if [ -z "${CHROMEOS_RELEASE_VERSION}" ]; then
+  echo -e "${RED}Only Chrome OS developer shell is supported by Chromebrew :/${RESET}"
+  exit 1
+fi
+
+# disallow non-stable channels Chromr OS
+if [ "${CHROMEOS_RELEASE_TRACK}" != 'stable-channel' ]; then
+  echo -e "${YELLOW}The beta, dev, and canary channel are unsupported by Chromebrew"
+  echo -e "Install anyway? [Y/n]"
+  echo -e "(This may cause major issues with your Chromebrew installation)${RESET}"
```

Comment:
It might be better to swap the 2 lines above so the cursor isn't below everything.

Reply:
We should also have a way to override this, with an env variable or whatever. We want to be able to dogfood on testing channels ourselves without huge hacks.

Reply:
Replaced `read` to variable